Update the Gov note for account management to include username/email (#28492)
DataDog/documentation: content/en/account_management/_index.md

      text: "Best practices for managing Datadog organizations at scale"
---
{{< site-region region="gov" >}}

> <div class="alert alert-warning">TThe Datadog for Government platform exclusively supports SAML or basic username/email and password authentication. Before configuring SAML authentication, you must establish at least one username/email and password account to ensure continued access during the setup process.

Reviewer: is it worth also advising here that we advise they set up MFA for their password accounts or that we support MFA so they can protect their password accounts

```suggestion
<div class="alert alert-warning">The Datadog for Government platform exclusively supports SAML or basic username/email and password authentication. Before configuring SAML authentication, you must establish at least one username/email and password account to ensure continued access during the setup process.
```

Author: @hay-sam  Yes! Updated to include MFA recommendation!

> <div class="alert alert-warning">TThe Datadog for Government platform exclusively supports SAML or basic username/email and password authentication. Before configuring SAML authentication, you must establish at least one username/email and password account to ensure continued access during the setup process.

Reviewer 2: ```suggestion
<div class="alert alert-warning">The Datadog for Government platform supports only SAML or basic authentication using a username/email and password. Before configuring SAML authentication, ensure that at least one username/email and password account is established to maintain access during the setup process.
```
